variables.tf

```
@@ -189,6 +189,12 @@ variable "skip_final_snapshot" {
   description = "Determines whether a final snapshot is created before deleting the cluster"
 }
 
+variable "snapshot_identifier" {
+  type        = string
+  default     = ""
```

Hi @rogiermccv, better to use `null` instead of an empty string here:

```suggestion
  default     = null
```